src/main/java/com/pusher/client/connection/websocket/WebSocketConnection.java

```diff
@@ -31,7 +30,9 @@
     private static final Gson GSON = new Gson();
 
     private static final String INTERNAL_EVENT_PREFIX = "pusher:";
-    static final String PING_EVENT_SERIALIZED = "{\"event\": \"pusher:ping\"}";
+    private static final String PING_EVENT_SERIALIZED = "{\"event\": \"pusher:ping\"}";
+    private static final int MAX_CONNECTION_ATTEMPTS = 6; //Taken from the Swift lib
```

Comment:
Perhaps `MAX_RECONNECTION_ATTEMPTS = 6 // Same as Swift lib`?

It's not just `CONNECTION` because this logic is only executed when there's a close.